global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "7.0.200",
+    "version": "8.0.100-rc.2.23502.2",
     "rollForward": "latestFeature",
     "allowPrerelease": false

```

Comment:
should `allowPrerelease` be set to `true`?

Reply:
yes indeed :) it should be for now.

fixed!